### PR TITLE
[Ready]docs about job's min resource

### PR DESCRIPTION
**Pull request by lowang-bh**
design for #2921

### CODE REVIEW

**Comment by lowang-bh** [comment]
| 3 | - | 1 | 3/200m |3/200m|
image
image

| 3 | 1 | - | 3/200m |3/250m|
image
image

| 2 | - | 1 | 2/100m |2/100m|
image
image

**Comment by hwdef** [comment]
I've been a bit busy lately and will probably review the two PRs you've recently @ me over the weekend

File: docs/design/min-avaiable-member-resource.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+now, job.MinAvailable <= sum(task.Replicas)
+
+- [x] minResource equals sum of resource among first jobMinAvailable, sorted by priority
+
+### 2.2 task's MinAvailable is not null
+
+job.MinAvailable >= sum(task.MinAvailable)
+
+- [ ] minResource equals sum of task.MinAvailable, then sum up resource of leftNumber(equals to job.MinAvailable - total(task.MinAvailable)) sorted by priority
```
Comment (Monokaix) [inline docs/design/min-avaiable-member-resource.md:91]: Do you mean choose a high priority task to add its resoucre to minResource when job.MinAvailable >= sum(task.MinAvailable)?

Comment (lowang-bh) [reply docs/design/min-avaiable-member-resource.md:91]: > Do you mean choose a high priority task to add its resoucre to minResource when job.MinAvailable >= sum(task.MinAvailable)?
Yes, the order to choose tasks is from high to low, and task number is job.MinAvailable - sum(task.MinAvailable).

**Comment by lowang-bh** [comment]
docs about merged PR #3057.
@Monokaix @william-wang

**Review by hwdef — COMMENTED** [review]
/lgtm

**Pull request merged**